docs/metrics/ncit.md

```diff
@@ -2,14 +2,14 @@
 
 **IRI:** http://purl.obolibrary.org/obo/mondo/sources/ncit.owl
 
-**Version IRI:** http://purl.obolibrary.org/obo/mondo/sources/2024-08-13/ncit.owl
+**Version IRI:** http://purl.obolibrary.org/obo/mondo/sources/2024-09-21/ncit.owl
 
 ### Entities and axioms
 
 | Metric | Value |
 | ------ | ----- |
 | Annotation properties | 12 |
-| Axioms | 623839 |
+| Axioms | 1328208 |
```

Comment:
Interesting increase.

Reply:
This large increase in NCIT has been consistent for the past month or so. I think it's because we've been doing builds on feature branches off of develop, but haven't actually committed any updated build outputs to develop for a while.